index.html

```diff
@@ -9284,9 +9345,9 @@ <h2 id="defining-a-valid-audioworkletprocessor">
                 The output audio buffer that is to be consumed by the user
                 agent. <code>outputs[n][m]</code> is a
                 <code>Float32Array</code> object containing the audio samples
-                for <code>m</code>th channel of <code>n</code>th output. While
-                the number of outputs is fixed at the construction, the number
-                of channels can be changed dynamically.
+                for <code>m</code>th channel of <code>n</code>th output. The
+                number of channels in the output can be dynamically changed
+                only when the node has single output.
```

Comment:
A link to the section describing this would be helpful.

Reply:
Rephrased it with a link.